apps/reaction/src/checkNodeVersion.cjs

```diff
@@ -4,8 +4,9 @@
 // this check to work on older Node versions that
 // do not support ES modules.
 
+import * as packageJson from "./package.json" assert { type: "json" };
```

Comment:
Some of these have `* as` and some files don't. I think it isn't necessary for JSON files, but not sure.

Reply:
@aldeed 
Yes, you are correct. The above ones did not throw any error while testing, but I got error when I made updates to CLI/Admin and used them together. I shall revert those two file updates [[checkNodeVersion.cjs & setupJestTests.js].

I also need to resolve the pnpm-lock.yaml conflict after changing the base branch

cc @zenweasel

Reply:
Changed only `setupJestTests.js`
`checkNodeVersion.cjs` restored back to the original 'require'